src/PsfFlux.cc

```diff
@@ -93,7 +93,9 @@ void PsfFluxAlgorithm::measure(afw::table::SourceRecord& measRecord,
                                    ->clippedTo(exposure.getMaskedImage().getMask()->getBBox()));
     }
     if (fitRegion.getArea() == 0) {
-        throw LSST_EXCEPT(MeasurementError, NO_GOOD_PIXELS.doc, NO_GOOD_PIXELS.number);
+        _flagHandler.setValue(measRecord, NO_GOOD_PIXELS.number, true);
+        _flagHandler.setValue(measRecord, FAILURE.number, true);
+        return;
```

Comment:
Shouldn't line 81 above also return? It looks like it sets the FAILURE flag and so is indicating that there is a real problem yet it is allowed to continue.

Reply:
Do do have a policy of algorithms attempting to produce the best measurement possible even when they know something is very wrong and set a flag to say so.  I suspect that's what's going on here.